.github/workflows/nightly.yml

```diff
@@ -121,7 +121,7 @@ jobs:
     needs: [build, binary, unit-tests]
     strategy:
       matrix:
-        k8s: [1.21.1, 1.20.7, 1.19.11, 1.18.19, 1.17.17, 1.16.15]
+        k8s: [1.21.1, 1.20.7, 1.19.11]
```

Comment:
```suggestion
        k8s: [1.21.2, 1.20.7, 1.19.11]
```
new minor version of 1.21

Reply:
no 1.22?

Reply:
@lucacome I'll send a separate PR. it will also change the version in the edge version

Reply:
@vepatel yep, no 1.22 for now